app/src/main/java/org/systers/mentorship/remote/BaseUrl.kt

```diff
@@ -16,10 +16,14 @@ object BaseUrl {
     private const val S3_BUCKET_NAME = "systers-mentorship/"
     private const val S3_PRODUCTION_DIRECTORY = "prod/"
     private const val S3_DEVELOPMENT_DIRECTORY = "dev/"
+    private const val LOCALHOST_URL = "10.0.2.2"
+    private const val LOCALHOST_PORT = ":5000"
 
     val apiBaseUrl: String
-        get() = if (BuildConfig.DEBUG) {
+        get() = if (BuildConfig.BUILD_TYPE == "debug"){
```

Comment:
Change request: This should not have been changed.

Reply:
This has been change because we have now 2 debug builds. If you want I can wrap it around BuildConfig.DEBUG. So now when you select `debug_localhost` from build variants you connect to your localhost

Reply:
Can you please explain what this is I didn't get the point. 

**My understanding** - 
There an be multiple flavours of an app. Each flavour will have a `debug` and `release` mode. If we are using the `debug` mode then `BuildConfig.DEBUG` will be `true` else in case of `release` mode we it is `false`.